CHANGELOG.md

```diff
@@ -9,8 +9,10 @@ version number is tracked in the file `VERSION`.
 ## Unreleased
 ### Changed
 - Small tidyups of environment collection module - PATCH
+- Also search ancestors of the working directory for a `floki.yaml` - MINOR
 ### Added
 - Forward host working directory as `FLOKI_HOST_WORKDIR` - MINOR
+- Add `-C <path>` option allowing floki to be run as if started at `<path>` - MINOR
```

Comment:
I wonder whether `-C` is a good choice of switch. Perhaps `-w` for working directory, and to provide some consistency with docker.

Reply:
I chose `-C` in common with `git`/`make`, but I imagine users are going to be more in the context of docker when running this tool. That said, docker's `-w` flag doesn't do the same thing - I'd be wary of confusing users who expect it to modify something in the guest. For that reason I'm inclined to leave it as `-C`, but I'd be happy to modify it if you disagree.

Reply:
See my comment below - we could cheekily sidestep this entirely if we leave `-C` out for now.

Reply:
As below, let's leave out `-C` for now.